pkg/library/flatten/flatten.go

```diff
@@ -272,3 +287,30 @@ func resolveElementByURI(
 	}
 	return dwt, nil
 }
+
+// canImportDW returns true if a DevWorkspace in dwNamespace is allowed to reference the provided DevWorkspaceTemplate
+// DevWorkspaces can by default only read DevWorkspaceTemplates in their own namespace, unless the DevWorkspaceTemplate
+// has the
```

Comment:
Looks like the comment wasn't finished :smile:

Reply:
Woopsie :D